### DISCUSSION TITLE
Fluent UI themed data table?

### DISCUSSION THREAD

**Discussion by TreyThomas93** [bdlukaa/fluent_ui#653] [Ideas]
Does anyone know if there will be fluent ui themed data tables in the future?

**Comment by Hopy-Jaden** [comment]
I guess no
https://github.com/bdlukaa/fluent_ui/issues/249

Here is a alternative solution I found but not yet tried before:
https://github.com/bdlukaa/fluent_ui/issues/420#issuecomment-1179688439
